src/components/LooMap/Markers.tsx

```diff
@@ -219,6 +198,7 @@ const MarkerGroup: React.FC<{
         const childCount = cluster.getChildCount();
         const children = cluster.getAllChildMarkers();
         const containedIds = children
+          // @ts-expect-error -- trust me, toiletId is there
           .map((child) => child.getIcon().options?.toiletId)
```

Comment:
Actually you can also just make TS happy with `flatMap`

```ts
const ids =  children.flatMap(child => {
  const id = child.getIcon().options?.toiletId;
  return typeof id === 'string' ? [id] : [];
})
```

Reply:
let's resolve this down the line. I want to refactor markers anyway